interactive_engine/executor/engine/pegasus/server/src/cluster/k8s.rs

```diff
@@ -0,0 +1,37 @@
+use std::net::SocketAddr;
```

Comment:
Copyright.

Reply:
done

Reply:
This file called `k8s.rs`. Donot think is appropriate.